builtin/providers/aws/resource_aws_app_cookie_stickiness_policy.go

```diff
@@ -90,7 +91,7 @@ func resourceAwsAppCookieStickinessPolicyRead(d *schema.ResourceData, meta inter
 
 	getResp, err := elbconn.DescribeLoadBalancerPolicies(request)
 	if err != nil {
-		if ec2err, ok := err.(aws.APIError); ok && ec2err.Code == "PolicyNotFound" {
+		if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "PolicyNotFound" {
```

Comment:
> All SDK methods will now return an awserr.Error instead of aws.APIError as a Go error.

Safe to leave for now, but going forward casting like this can go away now, and just directly start using the error:

```go
 getResp, awsErr := elbconn.DescribeLoadBalancerPolicies(request)
 if awsErr != nil {
   if awsErr.Code() == "PolicyNotFound" {
    ....
```